.travis.yml

```diff
@@ -23,7 +23,7 @@ install:
 - npm ci # installs from package-lock.json
 before_script:
 - SOLR_VERSION=$SOLR_VERSION SOLR_CORE=$SOLR_CORE bash ci/config_solr.sh
-- cp -r solr_conf downloads/solr-${SOLR_VERSION}/server/solr/configsets/sandco  # copy in local configset
+#- cp -r solr_conf downloads/solr-${SOLR_VERSION}/server/solr/configsets/sandco  # copy in local configset
```

Comment:
I like moving this to `config_solr.sh`. Do we need to keep the comment?

Reply:
Nope, don't need the comment now that we know this approach works. Do you think the configset name / path should be an environment variable, or does it matter? (Maybe not important until if/when we generalize the config solr script?)